### PR TITLE
Adds custom Unstable textbox 

### PR DESCRIPTION
**Pull request by DiogoLG**
We can add a custom message for FAILED, SUCCESS or ABORTED builds.

This MR adds the option to add a custom message for UNSTABLE builds as well. 

Does not change the current behavior to consider a build as success or failed (or +1/-1 a MR).

### CODE REVIEW

**Comment by DiogoLG** [comment]
@omehegan could you take a look at this?

**Comment by omehegan** [comment]
Yep, as soon as I have some time :) Hopefully this week.

File: src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java [hunk 1]
```diff
@@ -75,6 +83,10 @@ public String getAbortNoteText() {
         return this.abortNoteText == null ? "" : this.abortNoteText;
     }
 
+    public String getUnstableNoteText() {
+        return this.unstableNoteText == null ? "" : this.unstableNoteText;
```
Comment (arielb2) [inline src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java:87]: When ternary is used and we have binary operators it is recommended to use parenthesis.
Ex.
<code>
   return (this.unstableNoteText == null) ? "" : this.unstableNoteText;
</code>

Comment (DiogoLG) [reply src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java:87]: I will update that, but I just follow the pattern that was in the code already

**Comment by omehegan** [comment]
This was released today in version 1.4.6 of the plugin.

**Pull request merged**